### ISSUE TITLE
No HV from bias-control-lt8362 boost converter

### ISSUE THREAD

**Issue by nobias** [OpenSiPM/sipm-bias-control#10]
Hi! Thank you for this nice open source design, it has been very instructive for us. We had a few sets of PCBs made and assembled, and are currently trying to get the bias-control-lt8362 board to behave, with the other two PCBs not yet connected. We've flashed bootloader and firmware and checked the various low voltage supplies. We also verified that the LDO_SHDN and SHDN outputs from the MCU as well as the VOU0 and VOUT1 lines from the MCP respond as expected to serial commands. All that looks good.

However, we can't get the HV output from the LT8362 to go up to the expected ~50V (as measured at test point HV1). That voltage remains at the Vin level of 5V, no matter what we do. We tried enabling/disabling the LT8362, and changing the Boost_Gain DAC output from the MCU across the full range, without effect. We measured the voltage at the FBX feedback input to the LT8362 to be ~0.25V, which is the voltage that's to be expected if BOOST_OUT is 5V. But now the LT8362 should regulate BOOST_OUT up, which is not happening.

We've verified things like resistor values and diode orientations, all looking good. So I have to admit that we're a bit clueless as to why the LT8362 isn't doing its thing. If you had any hints as to what we could test, we'd be very grateful. Disclaimer - we're physicists who can mostly read a schematic, but not real electrical engineers, so we might be missing something basic here. Any hints appreciated!

Best,
Tobias

**Comment by mgiacomelli** [comment]
Hi Tobias,

From your description though you've probed most of what I was going to suggest.  Clearly the boost converter is not switching, and since the diode is conducting, you have it in the right orientation.  My next guess would be that something on the LT8362 is not soldered right.  Do you get ~3.2V on capacitor C3 (LT8362 internal supply voltage)?  Have you tried reflowing the whole chip?  I've seen this once or twice where the reflow temperature didn't get hot enough to solder the bottom ground connection ('pin' 17), in which case the chip never powers up.  

The current version is a little annoying to troubleshoot because I didn't put enough test points on the board, which I need to fix.  I was going to sit down and write up a sequence to test and add corresponding test points in the event that the board doesn't work.  

Mike

**Comment by nobias** [comment]
Hi Mike,

Thanks for your quick reply. We do see the 3.2V on C3, so that checks out. We did a first quick reflow attempt that didn't lead to an improvement, but we'll experiment some more with that to make sure we properly reflowed the big GND pad. 

So this is a bit weird. All looks quite good as far as I can tell. We'll stare some more at the boards and keep you posted.

Best,
Tobias

**Comment by nobias** [comment]
Hi Mike,

Some progress: Inspired by [this forum post](https://ez.analog.com/power/f/q-a/164002/lt8362-mode-pin-problem) we tried tying the SYNC/MODE pin to GND using a jumper wire, rather than leaving it floating. This fixed the issue: We got 52V at the HV1 test point. So there might have been some EMI affecting the SYNC/MODE input. Grounding that input switches to a different switching timing mode than when left floating ("burst" rather than "pulse-skip"), but we get a HV output, so I'm going to take it. 

We also tried connecting SYNC/MODE to 3.3V, which should select the "pulse-skip/SSFM" mode, but in that case, the output at HV1 remains at 5V, so no switching. No idea why.

Just one follow-up question: We're now seeing 52V at HV1, but we would have expected 55V, according to the equations in the schematic and datasheet, given the resistor values. This doesn't sound so far off, but we'd actually like to reach a bias voltage for the SiPM of ~58V. That's because we're using a different SiPM model than the one in your design, which requires a higher voltage. So we tried exchanging R6 and R16 for resistors with lower values (45kOhm), which should in theory have extended the reach of BOOST_OUT all the way up to the maximum 60V, and also the output of the regulator, depending on the GAIN and BOOST_GAIN voltages from the DACs. However, the LT8362 never delivers more than 52V, regardless of these resistor values. Do you have any clue as to why that might be? Can this have something to do with the SYNC/MODE configuration? We tested this both with an open load and with the other two boards and SiPM connected, without much difference.

Please let me know if I ask for too much time, I understand that open source designs don't come with a support contract... But if you had some quick hints, that would certainly be very valuable.

Best,
Tobias

**Comment by mgiacomelli** [comment]
Hi Tobias,

Regarding the regulator voltage, probably you realize this and have checked, but it can be controlled digitally via the DAC pin on the SAMD chip.  It should vary between 55V and 0V from the DAC down to about 8V at 3.3V.  So just to confirm, you have set the DAC voltage to 0V ("dac 0")?  The default is about 53V I think.  Although changing the resistors should change that value too...

Regarding higher output voltage, the LT8362 is rated for 60V output, but it is pin compatible with the LT8361 which can do up to 100V.  We swapped out the regulator and R6/R16 resistors for 39k and then ran the S13360 at 65V with no issues.  One student also mixed up the chips and ran the 8362 at 65V for some time with no apparent problem, so it seems fairly hard to break.  

I'm not sure what to think about the SYNC/MODE pin.  We have put together at least 15 of these boards, most by undergraduate students learning to solder, and so I've fixed a lot of them that didn't work initially.  You could be unlucky, but I've never had to touch that pin before.  The low maximum voltage with the changed resistors still makes me think a bad solder joint somewhere.  Have you tried taking the LT chip off, cleaning the solder from the pads, and reflowing back down?  That has fixed a student's board that looked well soldered, but inexplicably wouldn't start switching. 

Mike

**Comment by nobias** [comment]
Alright, we finally found the time to properly de- and resolder the LT8361, and now it works (without jumpering the SYNC/MODE pin to ground). I had to push the IC down a little bit while the solder was molten, to make sure that the big GND pad attaches really closely. Thank you very much for your kind support! We'll try the LT8362 and some higher voltages, to get some more gain out, but we're seeing photon clicks, so that's quite amazing!
